lib/modules/userTagger.js

```diff
@@ -940,25 +926,68 @@ modules['userTagger'] = {
 			}
 		}
 	},
-	ignoreComment: function(thisAuthorObj, thisAuthor){
-		var thisComment = thisAuthorObj.parentNode.parentNode.querySelector('.usertext');
-		if (thisComment) {
-			var showLink = document.createElement('a');
-			showLink.textContent = 'show anyway?';
-			showLink.setAttribute('href','#');
-			showLink.addEventListener('click', function(e) {
-				$(this).parent().html($(this).parent().attr('data-original')).removeClass('ignoredUserComment');
+	regexes: {
+		front: /^https?:\/\/(?:[\-\w\.]+\.)?reddit\.com\/?$/i
```

Comment:
Also you probably want to allow a query string or hash, i.e. `<snip>\.com\/?(?:[?#]|$)/i`